quantized_mesh_encoder/constants.py

```diff
@@ -1,20 +1,6 @@
 import numpy as np
 
-# From https://github.com/loicgasser/quantized-mesh-tile/blob/master/quantized_mesh_tile/llh_ecef.py
-# Constants taken from http://cesiumjs.org/2013/04/25/Horizon-culling/
-RADIUS_X = 6378137.0
-RADIUS_Y = 6378137.0
-RADIUS_Z = 6356752.3142451793
-
-# Stolen from https://github.com/bistromath/gr-air-modes/blob/master/python/mlat.py
-# WGS84 reference ellipsoid constants
-# http://en.wikipedia.org/wiki/Geodetic_datum#Conversion_calculations
-# http://en.wikipedia.org/wiki/File%3aECEF.png
-WGS84_A = RADIUS_X  # Semi-major axis
-WGS84_B = RADIUS_Z  # Semi-minor axis
-WGS84_E2 = 0.0066943799901975848  # First eccentricity squared
-WGS84_A2 = WGS84_A ** 2  # To speed things up a bit
-WGS84_B2 = WGS84_B ** 2
+WGS84 = { "a": 6378137.0, "b": 6356752.3142451793 }
```

Comment:
Are ellipsoids defined by two or three axes? I thought in general ellipsoids needed to be defined by three axes, and it just happened that the WGS84 reference ellipsoid had the X and Y radii equal. Maybe the code assumes X and Y radii will be equal?

Reply:
I went with a,b style definitions because the non-earth planetary data I've worked with has also consistently been defined in WKT/proj4 using semi-major and semi-minor axis (or flattening) style ellipsoid definitions. [IAU2015 definitions](https://github.com/USGS-Astrogeology/GDAL_scripts/tree/master/OGC_IAU2000_WKT_v2/IAU2015_prjs)

X,Y,Z gives more flexibility, though to properly support we may need a `to_ecef()` method that takes into account all three axes.